charts/consul/values.yaml

```diff
@@ -1934,7 +1934,9 @@ connectInject:
     # of the same name as their k8s namespace, optionally prefixed if
     # `mirroringK8SPrefix` is set below. If the Consul namespace does not
     # already exist, it will be created. Turning this on overrides the
-    # `consulDestinationNamespace` setting.
+    # `consulDestinationNamespace` setting. If mirroring is enabled, avoid creating any Consul 
+    # resources in the following Kubernetes namespaces, as Consul currently reserves these namespaces
+    # for system use: "system", "universal", "operator", "root". 
```

Comment:
should put this on all `mirroringK8S` stanzas (sync and api gateway)

Reply:
Thanks!